src/api/spec/features/webui/packages_spec.rb

```diff
@@ -115,6 +117,30 @@
     expect(find('#flash-messages')).to have_text('Package was successfully removed.')
   end
 
+  describe 'existing requests' do
+    let(:source_project) { create(:project_with_package, name: 'source_project') }
+    let(:source_package) { source_project.packages.first }
+    let(:target_project) { user.home_project }
+    # let(:target_package) { package.name }
```

Comment:
:eyes:

Reply:
could you please re-review it @DavidKang

Reply:
Please, remove this comment and also [vcr comment](https://github.com/openSUSE/open-build-service/pull/5527/files#diff-fafd54efb3601a2010161498e1fefb6fR4)